README.md

```diff
@@ -11,7 +11,7 @@ You can use this package to verify your API usage is compatible with a range of
 ```ts
 /// <reference types="@nextcloud/typings" />
 
-declare var OC: Nextcloud.v19.OC | Nextcloud.v20.OC | Nextcloud.v21.OC | Nextcloud.v22.OC;
+declare var OC: Nextcloud.v20.OC | Nextcloud.v21.OC | Nextcloud.v22.OC | Nextcloud.v23.OC;
```

Comment:
We can remove v20 here since only the last 3 versions are supported https://docs.nextcloud.com/server/latest/admin_manual/release_schedule.html

Reply:
Sure, I just kept the same number.